libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo/network/http/HttpNetworkTransport.kt

```diff
@@ -218,17 +222,16 @@ private constructor(
               else -> null
             }
           } else {
```

Comment:
Is there a way to test from the response whether it supports the incremental delivey protocol? Is there something in `response.headers["content-type"]` that signals what protocol was used by the server?